lib/elixir/lib/macro.ex

```diff
@@ -153,7 +153,7 @@ defmodule Macro do
     raise ArgumentError, bad_pipe(expr, call_args)
   end
 
-  # Without this, `Macro |> Env == Macro.Env`.
+  # Without this: "Macro |> Env == Macro.Env"
```

Comment:
Yep, the comment was meant to say that "Without this code, this happens". I would leave `,` and `.` there. (btw, I just remembered that I'm the one that wrote that comment! :)).